configs/forerunnerI_gnu.config

```diff
@@ -0,0 +1,53 @@
+# forerunnerI
+### Caution!! Load modules under module_CALAB on TwnIII first before compiling, such that FFTW3_PATH, MPI_PATH, HDF5_PATH and GSL_PATH can be found:
+###     module use  /home/d07222009/module_CALAB
+###     module load gnu_13.2.0/gcc/13.2.0 gnu_13.2.0/fftw/3.3.10 gnu_13.2.0/gsl/2.8.0 gnu_13.2.0/hdf5/1.14.4 gnu_13.2.0/openmpi/5.0.0  gnu_13.2.0/openucx/1.18.0
+###
+CUDA_PATH       
+FFTW2_PATH      
+FFTW3_PATH      $(FFTW3_PATH)
+MPI_PATH        $(MPI_PATH)
+HDF5_PATH       $(HDF5_PATH)
+GRACKLE_PATH
+GSL_PATH        $(GSL_PATH)
+LIBYT_PATH
+CUFFTDX_PATH    
+
+# compilers
+CXX     g++-13.2.0
+CXX_MPI mpicxx
+
+# flags
+CXXFLAG -g
+CXXFLAG -O3
+#CXXFLAG -std=c++11
+#CXXFLAG -Ofast
+CXXFLAG -Wall
+CXXFLAG -Wextra
+CXXFLAG -Wno-unused-variable
+CXXFLAG -Wno-unused-parameter
+CXXFLAG -Wno-maybe-uninitialized
+CXXFLAG -Wno-unused-but-set-variable
+CXXFLAG -Wno-unused-function
+CXXFLAG -Wno-unused-result
+CXXFLAG -Wno-implicit-fallthrough
+CXXFLAG -Wno-parentheses
+CXXFLAG -Wno-unknown-pragmas
+
+OPENMPFLAG -fopenmp
+
+LIBFLAG
+
+NVCCFLAG_COM -O3
+#NVCCFLAG_COM -use_fast_math
+NVCCFLAG_FLU -Xptxas -dlcm=ca -prec-div=false -ftz=true
+NVCCFLAG_POT -Xptxas -dlcm=ca
+
+# for debugging
+#CXXFLAG -fstack-protector-all
+#CXXFLAG -fstack-protector-strong
+#CXXFLAG -fsanitize=undefined -fsanitize=address
+#LIBFLAG -fsanitize=undefined -fsanitize=address
+
+# gpu
+GPU_COMPUTE_CAPABILITY 860    # 3080 Ti
```

Comment:
To make sure, is this needed if `forerunnerI` doesn't have GPU?

Reply:
```
# gpu
GPU_COMPUTE_CAPABILITY 860    # 3080 Ti
```
has been removed, though it will cause warning messages when running `configure.py`.